server/http_handler.go

```diff
@@ -227,7 +227,18 @@ func (t *tikvHandlerTool) getMvccByIdxValue(idx table.Index, values url.Values,
 	if err != nil {
 		return nil, err
 	}
-	return &helper.MvccKV{Key: strings.ToUpper(hex.EncodeToString(encodedKey)), RegionID: regionID, Value: data}, err
+	idxData := &helper.MvccKV{Key: strings.ToUpper(hex.EncodeToString(encodedKey)), RegionID: regionID, Value: data}
+	tablecodec.IndexKey2TempIndexKey(idx.Meta().ID, encodedKey)
+	data, err = t.GetMvccByEncodedKey(encodedKey)
```

Comment:
Should we check the index's backfill state?

Reply:
No, it may be public, but we still need to get the temp index.